pkg/cmd/root.go

```diff
@@ -45,6 +44,8 @@ func init() {
 
 	RootCmd.PersistentFlags().StringVarP(&basePath, "path", "p", "/var/lib/dvol/volumes",
 		"The name of the directory to use")
+	RootCmd.PersistentFlags().StringVar(&outputFormat, "output-format", "text",
+		"Output format: either 'text' or 'json'")
```

Comment:
This option is global but it only does anything with 'log'? Is it planned to introduce json formatted output to all commands? If so we should probably file an issue for it.